tool/gravity/cli/plan.go

```diff
@@ -231,4 +228,21 @@ func tryReconcilePlan(ctx context.Context, localEnv, updateEnv *localenv.LocalEn
 	return reconciledPlan, nil
 }
 
-const recoveryModeWarning = "Failed to retrieve plan from etcd, showing cached plan. If etcd went down as a result of a system upgrade, you can perform a rollback phase. Run 'gravity plan --repair' when etcd connection is restored.\n"
+func getPlanFromWizard(opKey ops.SiteOperationKey) (*storage.OperationPlan, error) {
+	wizardEnv, err := localenv.NewRemoteEnvironment()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	if wizardEnv.Operator == nil {
+		return nil, trace.NotFound("no operation plan")
```

Comment:
Nit: I think the message here is a bit misleading as we didn't actually check for plan. I'd rather say why the operator is nil here.

Reply:
That's a good question - I have no idea how the operator could be nil after a successful call to `NewRemoveEnvironment`.